src/notifications/footer/FooterNotificationItems.tsx

```diff
@@ -0,0 +1,82 @@
+import { useAtomValue } from 'jotai';
+import { useCallback, useEffect } from 'react';
+import { VscBell, VscBellDot, VscError, VscInfo, VscWarning } from 'react-icons/vsc';
+
+import {
+  listNotificationsAtom,
+  notificationsPopupAtom,
+  notificationsTypeStatsAtom,
+} from '../../atoms/notificationsState';
+import { FooterItem } from '../../components/footer/FooterItem';
+import { cx } from '../../lib/cx';
+import { clearNotifications, hideNotifications, showNotifications } from '../notifications';
+import { NotificationItemType } from '../types';
+import { NotificationsPopup } from './NotificationsPopup';
+
+export function FooterNotificationItems() {
+  const notifications = useAtomValue(listNotificationsAtom);
+  const stats = useAtomValue(notificationsTypeStatsAtom);
+
+  const notificationsPopup = useAtomValue(notificationsPopupAtom);
+
+  useEffect(() => {
+    if (notifications.length > 0 && notifications[0].type === 'error') {
+      showNotifications('error');
+    }
+  }, [notifications]);
```

Comment:
This is the UX in VS Code and I think it would be usefull. Will keep it for now and then we can hide if is anying.

Reply:
My main concern is that we have errors when switching between editors, because of TipTap (which by the way has been fixed in a recent PR: https://github.com/ueberdosis/tiptap/pull/4000), so the the drawer will open a lot